src/main/java/org/kuali/kfs/sys/cache/CacheConfiguration.java

```diff
@@ -205,4 +212,10 @@ public RedisCacheManager cacheManager(
 
         return redisCacheManager;
     }
+    
+    protected String getEnvironmentPrefix() {
```

Comment:
Instead of adding this separate method to grab the config property from the Configuration Service, you could try injecting it as a parameter in the `cacheManager()` method. That method (along with others in this class) should have examples of how to use the `@Value` annotation for injecting property values as method parameters.